Step 1

BaggioGiacomo opened the pull request.

BaggioGiacomo committed:
Creazione app step1

Prova commit

BaggioGiacomo committed:
Fine step 1

Eliminato tutto il codice in pages/index.js e sostituito con un hello world

BaggioGiacomo committed:
Update index.js

dhmacs commented on step1/styles/Home.module.css:
```diff
@@ -0,0 +1,122 @@
+.card {
+  margin: 1rem;
+  flex-basis: 45%;
+  padding: 1.5rem;
+  text-align: left;
+  color: inherit;
+  text-decoration: none;
+  border: 1px solid #eaeaea;
+  border-radius: 10px;
+  transition: color 0.15s ease, border-color 0.15s ease;
+}
+
+.card:hover,
+.card:focus,
+.card:active {
+  color: #0070f3;
+  border-color: #0070f3;
+}
+
+.card h3 {
+  margin: 0 0 1rem 0;
+  font-size: 1.5rem;
+}
+
+.card p {
+  margin: 0;
+  font-size: 1.25rem;
+  line-height: 1.5;
+}
+
+.logo {
+  height: 1em;
+}
+
+@media (max-width: 600px) {
+  .grid {
+    width: 100%;
+    flex-direction: column;
+  }
+}
```
Please remove all unused styles

dhmacs reviewed (changes requested):
The first step overall looks pretty good. Please address the comment I've added. Moreover move the code from `/step1` folder into the root directory (don't override the readme containing instructions).

> **NOTE**: By convention git branch names should follow kebab case (e.g. `step-1`). Don't bother to rename `Step1`, just follow this conventions from now on

BaggioGiacomo committed:
Step1 final check

I've deleted all unused styles and i've brought step1 folder's files on the main directory

dhmacs reviewed (approved):
LGTM 👍
